src/DashboardUI/src/pages/admin/AdminOrganizationUsersPage.tsx

```diff
@@ -61,23 +67,86 @@ export function AdminOrganizationsUsersPage() {
     }
   };
 
+  const openRemoveUserModal = (userId: string) => {
+    setShowRemoveUserModal(!showRemoveUserModal);
+    setRemoveUserId(userId);
+  };
+
+  const closeRemoveUserModal = () => {
+    setShowRemoveUserModal(false);
+    setRemoveUserId('');
+  };
+
   return (
     <>
       <div className="overflow-y-auto h-100">
         <div className="m-3">
           {pageTitle()}
 
-          <div className="d-flex justify-content-between my-4">
-            <h2>Users</h2>
+          <div className="d-flex justify-content-between align-items-center my-4">
+            <h2 className="fs-5">Users</h2>
             {addUserButton()}
           </div>
 
           <TableLoading
             isLoading={organizationListLoading || organizationUsersListLoading}
             isErrored={organizationListErrored || organizationUsersListErrored}
           >
-            (<pre>OrganizationUsers: {JSON.stringify(organizationUsersListResponse)}</pre>)
+            <table className="w-100">
+              <thead>
+                <tr>
+                  <th className="px-2 py-1 border-bottom">User</th>
+                  <th className="px-2 py-1 border-bottom">Role</th>
+                  <th className="px-2 py-1 border-bottom">Email</th>
+                  <th className="px-2 py-1 border-bottom">User ID</th>
+                  <th className="px-2 py-1 border-bottom"></th>
+                </tr>
+              </thead>
+              <tbody>
+                {organizationUsersListResponse?.users &&
+                  organizationUsersListResponse?.users.length > 0 &&
+                  organizationUsersListResponse?.users.map((user) => (
+                    <tr key={user.userId}>
+                      <td key={`${user.userId}-name`} className="px-2 py-1 border-bottom">
+                        {user.lastName}, {user.firstName}
+                      </td>
+                      <td key={`${user.userId}-role`} className="px-2 py-1 border-bottom">
+                        {RoleDisplayNames[user.role]}
+                      </td>
+                      <td key={`${user.userId}-email`} className="px-2 py-1 border-bottom">
+                        <Button variant="link" href={`mailto:${user.email}`} className="px-0 user-email-button-text">
+                          {user.email}
+                        </Button>
+                      </td>
+                      <td key={`${user.userId}-userID`} className="px-2 py-1 border-bottom">
+                        {user.userName}
+                      </td>
+                      <td key={`${user.userId}-remove`} className="px-2 py-1 border-bottom text-center">
+                        <Button
+                          variant="link"
+                          className="px-3 py-1 remove-user-button-text"
+                          onClick={() => openRemoveUserModal(user.userId)}
+                        >
+                          Remove from Org
```

Comment:
Let's spell it out. I think we have plenty of room
```suggestion
                          Remove from Organization
```

We have other places where we spell it out
<img width="316" alt="image" src="https://github.com/user-attachments/assets/1c86c77e-a6b6-4d2e-acb8-dd7d44392e6d" />